src/rest_parser.cpp

```diff
@@ -84,7 +85,27 @@ bool RestParser::parseSpecialInput(rapidjson::Value& doc, tensorflow::TensorProt
     return false;
 }
 
+bool isBinary(const rapidjson::Value& value){
+    if(value.IsObject())
+    {
+        if(value.HasMember("b64")){
```

Comment:
What if there are more entries in the map apart from "b64"? 
Validation will pass, but what with additional data?